build-system/tasks/runtime-test/helpers.js

```diff
@@ -180,27 +180,27 @@ function karmaBrowsersReady_() {
  * @private
  */
 function karmaRunStart_() {
-  log(green('Running tests locally...'));
+  log(
+    green('Transforming tests with'),
+    cyan('esbuild'),
+    green('and'),
+    cyan('babel') + green('...')
+  );
 }
 
 /**
  * Creates and starts karma server
- * @param {!Object} configBatch
- * @param {function()} runCompleteFn a function to execute on the
- *     `run_complete` event. It should take two arguments, (browser, results),
- *     and return nothing.
+ * @param {!Object} config
  * @return {!Promise<number>}
  */
-async function createKarmaServer(
-  configBatch,
-  runCompleteFn = reportTestRunComplete
-) {
-  let resolver;
+async function createKarmaServer(config) {
+  let resolver, browsers_, results_;
   const deferred = new Promise((resolverIn) => {
```

Comment:
fyi: Have you seen the `Deferred` class? Is it okay to use `src/` code within build-system? It is a useful utility for situations like this one where you want the resolve/reject/promise objects

Reply:
So far, we've avoided using source code in `build-system/`. Not opposed to adding a util that does this more elegantly. (Via a follow up, since there are several such `resolver`/`deferred` patterns to replace.)